### PR TITLE
P9 EEEVIIILE

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request
P9 "Evil" Bluecross gun. Uses your health and max-health to fire it. EEEEVIL!!!
image

Based off the
https://user-images.githubusercontent.com/47883419/167241631-69f73c43-8232-44f7-be1f-b3b8537e7301.mp4
<hr>

## Changelog
:cl:
add: Adds the P9 "Evil" Bluecross SMG
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/oddity_items.dm [hunk 1]
```diff
@@ -299,6 +299,41 @@
...
+			An odd looking tool-made smg of sorts, made completely out of stamped metal and hatred. You wonder by looking at this how many people have used this worn weapon of war. \
+			Looking at it long enough appears to make you see red.. feeling as if its draining your life force just to fire it! Let the hatred RISE!"
+	icon = 'icons/obj/guns/energy/p9evil.dmi'
+	icon_state = "evil"
+	item_state = "evil"
+	origin_tech = list(TECH_ILLEGAL = 5)
+	matter = list(MATERIAL_PLASTEEL = 6, MATERIAL_STEEL = 25)
+	price_tag = 2500
+	projectile_type = /obj/item/projectile/beam/weak/pistol_35
+	charge_cost = 999999999999999999999999999999999999
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:314]: 1000000000000000000000000000000000000
```suggestion
	charge_cost = 1000000000000000000000000000000000000
```

File: code/modules/projectiles/guns/oddity_items.dm [hunk 2]
```diff
@@ -299,6 +299,41 @@
...
+	penetration_multiplier = 1.2
+	recoil_buildup = 2
+	one_hand_penalty = 10
+	gun_tags = list(GUN_ENERGY, GUN_LASER, GUN_SILENCABLE)
+	init_firemodes = list(
+		SEMI_AUTO_NODELAY,
+		BURST_3_ROUND,
+		FULL_AUTO_400
+		)
+	serial_type = "BlueCross"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:328]:
```suggestion
	serial_type = "Crimson Cross"
```

File: code/modules/projectiles/guns/oddity_items.dm [hunk 3]
```diff
@@ -299,6 +299,41 @@
 	gun_tags = list(GUN_PROJECTILE, GUN_CALIBRE_35, GUN_SILENCABLE, GUN_MAGWELL)
 	serial_type = "BlueCross"
 
+/obj/item/gun/energy/lasersmg/p9evil
+	name = "P9 \"Evil\" smg"
+	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a crimson cross, these weapons are known to vanish and reappear when left alone. \
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:304]:
```suggestion
	desc = "An anomalous weapon created by rivals of the unknown person(or group?) of the bluecross, their work marked by a crimson cross, these weapons are known to vanish and reappear when left alone. \
```

**Comment by cdb-is-not-good** [comment]
Funny gun, but run this by Pinkest since it establishes the existence of Red Cross :p

**Pull request merged**